docs/community/running-tests.md

```diff
@@ -1,42 +1,55 @@
-| `REGIONS` | no      | When you want to deploy to a randomly selected region from a known-working set of regions. E.g., `REGIONS=westus2,westeurope,canadacentral`. Required if `LOCATION` is empty; not required if `NAME` is provided. |
-| `CLUSTER_DEFINITION` | yes      | The API model to use as cluster configuration input for creating a new cluster. E.g., `CLUSTER_DEFINITION=examples/kubernetes.json`. Not required if `NAME` is provided. |
-| `CLEANUP_ON_EXIT` | no      | Delete cluster after running E2E. E.g., `CLEANUP_ON_EXIT=true`. Default is false. |
-| `CLEANUP_IF_FAIL` | no      | Delete cluster only if E2E failed. E.g., `CLEANUP_IF_FAIL=false`. Default is false. |
-| `STABILITY_ITERATIONS` | no      | How many basic functional cluster tests to run in rapid succession as a part of E2E validation. This is useful for simulation continual usage of basic cluster reconciliation functionality (schedule/delete a pod, resolve a DNS lookup, etc). E.g., `STABILITY_ITERATIONS=100`. Default is 3. |
-| `TIMEOUT` | no      | How much timeout tolerance for tests? Decrease timeout tolerance to do performance-type tests, increase to allow for more operational variability and possibly reduce flakes. E.g., `TIMEOUT=10m`. Default is 20m. |
-| `LB_TIMEOUT` | no      | How much timeout tolerance for Load Balancer tests? Decrease timeout tolerance to do performance-type tests, increase to allow for more operational variability and possibly reduce flakes. E.g., `LB_TIMEOUT=5m`. Default is 20m. |
-| `GINKGO_FAIL_FAST` | no      | Stop the suite right after the first failure? E.g., `GINKGO_FAIL_FAST=false`. Default is true. |
-| `GINKGO_FOCUS` | no      | Regular expression string to pass to test runner to run only a subset of tests that match the regular expression. E.g., `GINKGO_FOCUS="should be able to produce working LoadBalancers"`. Only works if `GINKGO_FAIL_FAST` is set to true. |
-| `GINKGO_SKIP` | no      | Regular expression string to pass to test runner to skip the subset of tests that match the regular expression. E.g., `GINKGO_SKIP="should be able to attach azure file"`. Only works if `GINKGO_FAIL_FAST` is set to true. |
-| `DEBUG_AFTERSUITE` | no      | Print out Kubernetes resources and logs after E2E suite. This is especially useful if you have to delete your cluster after running the test and you wish to debug at a later time. E.g., `DEBUG_AFTERSUITE=true`. Default is false. |
+# Running Tests
+
+If you are an AKS Engine developer, running local E2E tests to validate changes can greatly increase iterative velocity.
+
+As mentioned briefly in the [developer guide](developer-guide.md), a `make` target is maintained to provide convenient shell invocation of the E2E test runner across for generic, configurable usage:
+
+```sh
+# build local a copy of aks-engine
+$ make build
```

Comment:
Maybe we should update the instructions to build/run everything in the dev container?
```bash
make dev
make
```

then run set env vars/run e2e tests?

Reply:
this set of commands will be the same if in a dev container or not I believe

Reply:
yes - but it will be more likely that the dev container will have all of the tools installed and will more likely result in users being successful.
Can we add or mention running `make dev` in addition to your changes?
Both seem like good additions.